puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/tarballsRelease.groovy

```diff
@@ -57,7 +57,10 @@ pipeline {
 void verify_tag(project, version) {
     dir(project) {
         git url: "https://github.com/theforeman/${project}", branch: 'develop'
-        sh "test \"$(git tag -l ${version} | wc -l)\" -eq 1"
+        def found_version = sh script: "git tag -l ${version} | grep ${version}", returnStdout: true
+        if (found_version.trim() != version) {
```

Comment:
This may be redundant. For me grep is sufficient to verify:
```
$ git tag -l 1.10.0
1.10.0
$ git tag -l 1.1
1.1
```

I think you can just rely on the exit code because of that. @evgeni can you confirm this?

Reply:
Updated, I think you are right. As per [doc](https://jenkins.io/doc/pipeline/steps/workflow-durable-task-step/#code-sh-code-shell-script), script returning with non-zero status will fail.

```sh
git tag -l  |  grep 1.17.0
1.17.0
echo $?
0

git tag -l  |  grep 1.20
echo $?
1
```